WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadHandler.java

```diff
@@ -630,6 +635,21 @@ public void onPostUploaded(OnPostUploaded event) {
         finishUpload();
     }
 
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN, priority = 9)
+    public void onPostChanged(OnPostChanged event) {
+        AppLog.e(T.API, "causeOfChange: " + event.causeOfChange);
```

Comment:
We might want to consider logging this with `info` severity, wdyt?

Reply:
Yeah I just noticed it in my other branch, I forgot to put a FIXME there, I'll remove that line

Reply:
Removed in 63a9b59 - ready for a second pass.